examples/knative/go/kn-go-echo/server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+func main() {
+	ctx := context.Background()
+	protocol, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Fatalf("creating protocol: %s", err.Error())
+	}
+
+	client, err := cloudevents.NewClient(protocol)
+	if err != nil {
+		log.Fatalf("creating client, %v", err)
+	}
+
+	log.Printf("listening on :8080\n")
+	log.Fatalf("start receiver: %s", client.StartReceiver(ctx, receive))
+}
+
+func receive(ctx context.Context, event cloudevents.Event) {
+	fmt.Printf("***cloud event***\n%s", event)
```

Comment:
Will this work nicely with different payload encoding schemes, e.g. XML, JSON, plain text?

Reply:
Not sure if my answer is what you're looking for. I think you're asking if the `fmt.Print(...)` will print all encoding schemes nicely. Looking at the [String() method on the Event type](https://github.com/cloudevents/sdk-go/blob/a302fc5e5e6a462237f6166905c725921d4da131/v2/event/event.go#L70), it seems like the cloud event SDK library only makes json look nice.

Reply:
yeah, you answered my question which was heading in the non-JSON direction, e.g. XML which is currently used in the Tanzu Sources for vSphere.

Reply:
So in case of XML the output would not be nice, right?

Reply:
I haven't tried it, but I'm guessing XML output will be ugly. 

I don't even know if the Cloud SDK http receiver would accept it because it seems I have to use `Content-Type: application/cloudevents+json`. Even `Content-Type: application/json` doesn't seem to work.